conda.recipe/build.sh

```diff
@@ -0,0 +1,24 @@
+
+git clone https://github.com/fabm-model/fabm.git
+git clone https://github.com/gotm-model/code.git gotm
+cd gotm && git submodule update --init --recursive && cd ..
+
+mkdir build_pyfabm && cd build_pyfabm
+cmake ../fabm/src/drivers/python -DFABM_ERSEM_BASE=.. -DCMAKE_INSTALL_PREFIX:PATH=$PREFIX 
+sed -i -e 's/--user//g' cmake_install.cmake
```

Comment:
@jimc101 I tried to change the install directory with cmake but it always sets the `--user` pip install flag. If you can think of a better way of getting rid of it than to remove it from the `cmake_install.cmake` file then let me know

Reply:
Ah, that will be why I couldn't see it - I always set `PYTHONNOUSERSITE=True` when starting conda in order to isolate the conda environment from anything installed in `~/.local`. It would be nice to be able to pass a flag to cmake specifying whether we want to use `--user` or not (default use it, I guess). Would this have to be done in FABM? @jornbr?

Reply:
Yes that's a FABM thing. I'm currently restructuring the pyfabm installation procedure for other reasons (FABM branch pyfabm_nd), but that will help here too: to install you'll no longer call cmake, but setup.py (which internally sets up build dirs and calls cmake as part of the "build_ext" step). Thus --user can be added or left out as desired. I'd suggest holding out for these changes on the FABM side for now.

Reply:
I just ran the above commands again. Everything installs fine and pyfabm is now found. However, when typing:

```bash
$ gotm -h
```
I get a `gotm: error while loading shared libraries: libnetcdff.so.7: cannot open shared object file: No such file or directory` error. This didn't happen before. Have you changed anything that might impact the installation of NetCDF shared libraries?

Reply:
I get the same error when running `$ fabm0d -h`.

Reply:
I will have a look

Reply:
Problem now fixed.